mavros/msg/ActuatorControl.msg

```diff
@@ -0,0 +1,12 @@
+# raw servo values for direct actuator controls
+# 0: roll
+# 1: pitch
+# 2: yaw
+# 3: throttle
+# 4: flaps
+# 5: spoilers
+# 6: airbrakes
+# 7: landing gear
+
```

Comment:
You should remove these mappings, as they are control-group specific. A link to the wiki page for PX4 control groups and mixing would be more useful here.